ab2d-fhir/src/test/java/gov/cms/ab2d/fhir/FhirVersionTest.java

```diff
@@ -97,6 +98,23 @@ void testGetClassName() {
     ReflectionTestUtils.setField(FhirVersion.R4, "classLocation", classLocation);
   }
 
+  @Test
+  void testGetClassFromName() {
+    assertEquals(
+      org.hl7.fhir.dstu3.model.Patient.class,
+      FhirVersion.STU3.getClassFromName("Patient")
+    );
+    assertEquals(
+      org.hl7.fhir.r4.model.Patient.class,
+      FhirVersion.R4.getClassFromName("Patient")
+    );
+
+    assertDoesNotThrow(() -> {
+      FhirVersion.R4.getClassFromName("doesNotExist");
+    });
+    assertNull(FhirVersion.R4.getClassFromName("doesNotExist"));
+  }
```

Comment:
Do we need to add similar assertions here for STU3? 

assertDoesNotThrow and assertNull?

Reply:
Yeah sure